ans2ask-ui/src/components/Home/Home.jsx

```diff
@@ -9,12 +9,16 @@ import QuestionsOrCourses from "../QuestionsOrCourses/QuestionsOrCourses";
 import Subjects from "../Subjects/Subjects";
 import "./Home.css";
 
+//Options
+const Options = {
+  question: 1,
+  course: 2    
+}
+
```

Comment:
We should define this enum in a helper/utils class, so we do not need to redefine this in all related classes

Reply:
done!